christfellowshipchurch/web-app-v2
# Discover back button

### About
This PR was made so that anyone who accessed a link to any content item on the discover page would be redirected to `/discover` whenever the back button was pressed.

### Test Instructions
1. Go to /discover/messages?id=47a5a31f61ac5a4fb65576d0d47564e0 on the CF website
2. Press the back button and you should be redirected to the discover page

### Screenshots
![Screenshot 2023-06-20 at 9 57 17 AM](https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/cd407635-11f6-4d7d-9f86-b72f13c7de9e)


### Closes Tickets
[CFDP-2605](https://christfellowshipchurch.atlassian.net/browse/CFDP-2605)



[CFDP-2605]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2605?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
